### ISSUE TITLE
Incorrect Order for Recipient/Organization in German Postal Format

### ISSUE THREAD

**Issue by davidpersson** [commerceguys/addressing#34]
When both recipient and org are present the german addresses seem incorrectly formatted. In Germany postal addresses have the organization come _before_ the recipient. 

![bildschirmfoto 2015-05-21 um 12 23 19](https://cloud.githubusercontent.com/assets/29702/7746658/82c76bbe-ffb4-11e4-99b2-98c69fb6e8e0.png)
[According to Deutsche Post](http://www.postadressglobal.com/de-de/country/deutschland/formatierung-und-strukturierung-deutscher-adressen/)

However it seems odd that Google might be wrong here:
https://github.com/commerceguys/addressing/blob/master/resources/address_format/DE.json
http://i18napis.appspot.com/address/data/DE

In case this is just me getting the semantics wrong, feel free to close the ticket. 
Thanks for the great library!

**Comment by jonathanmaron** [comment]
I can confirm this: The company name comes on line 1 and the recipient on line 2, when printing an address block in Germany. Just yesterday, did I notice that order is incorrect. Is Germany unique in this regard?

**Comment by bojanz** [comment]
A grep through the dataset shows 28 countries that put the organization on the first line, including Austria, Belgium, France, Netherlands, Switzerland.
So, Germany is not unique here. The error in Google's dataset is probably caused by UPU being unclear about this: http://www.upu.int/fileadmin/documentsFiles/activities/addressingUnit/deuEn.pdf

I've opened a bug report in the libaddressinput issue queue: https://github.com/googlei18n/libaddressinput/issues/83

I've created a lot of issues in their queue, and they're being resolved very slowly (one in the past 6 months) so I'm going to start maintaining our own patch against the dataset, and include this fix.
